static/css/searchPage.css

```diff
@@ -1,32 +1,32 @@
-body {
-  font-family: Helvetica, sans-serif;
-  margin: 0px;
+body{
+    font-family: Helvetica, sans-serif;
+    margin: 0px;
 }
 
 /* Banner at top of page contains logo */
-#header-banner {
-  border: 1px solid #ccc;
-  background-color: whitesmoke;
-  width: 100%;
+#header-banner   {
+    border: 1px solid #ccc;
+    background-color: whitesmoke;
+    width: 100%; 
 }
 
 /* Logo at the top left corner */
-#logo {
-  display: inline-block;
-  width: 130px;
-  height: 90px;
+#logo   {
+    display: inline-block;
+    width:130px;
+    height:90px;
 }
 
 /* Input form i.e for name, location, etc. */
-#input-form {
-  margin-left: auto;
-  margin-right: auto;
-  width: 50%;
-  text-align: left;
-  background-color: #e2f1ea;
-  border-radius: 0.5rem;
-  box-shadow: 5px 8px 10px #d1d1d1;
-  padding: 25px;
+#input-form{
+    margin-left:auto;
+    margin-right: auto;
+    width: 50%;
+    text-align: left;
+    background-color: #e2f1ea;
+    border-radius: 0.5rem;
+    box-shadow: 5px 8px 10px #d1d1d1;
+    padding: 25px; 
```

Comment:
I'm guessing (not sure, but high confidence) that the merge conflicts were resolved by accepting the non-formatted CSS prior to the the merging of the PR that combined backend/frontend.

To clarify, my PR that merged the backend & frontend also included formatting changes on every file because I ran `npm run format` before merging in the PR. (I probably should have extracted formatting into a separate PR).

This probably caused merge conflicts when you tried to sync this PR with the new codebase because the CSS in your PR is unformatted whereas the CSS in the new codebase is formatted.

It looks like the merge conflict was resolved by accepting the non-formatted CSS (hence the trailing whitespace on this line, and the lack of spaces between the id name and the left bracket, and the lack of space between `margin-left` and `auto`.

2 possible solutions here are:
1. doing a hard reset to remove the last 2 commits that resolved merge conflicts and re-trying the merge (or potentially rebasing) in order to get the correct set of changes (formatted CSS). This would be harder but lead to a cleaner commit history.
2. Run `npm run format` on the codebase and then add a new commit called "formatting" so everything has the proper formatting. (much easier, adds an extra commit -- which is definitely ok!)

Let's make sure the PR is merged in once all the files are formatted properly. :smile:

Reply:
Instead of trying to resolve conflicts, I think its better I run `npm run format` on my end. Just to avoid complications